python/fusion_engine_client/messages/solution.py

```diff
@@ -287,30 +302,36 @@ def unpack(self, buffer: bytes, offset: int = 0) -> int:
         offset += self.p1_time.unpack(buffer, offset)
         offset += self.gps_time.unpack(buffer, offset)
 
-        offset += self.last_differential_time.unpack(buffer, offset)
-
-        (self.reference_station_id,
+        (self.leap_second, self.num_svs,
```

Comment:
Even though we didn't resize this struct, should we check the message version to avoid setting these values to random garbage, and just invalidate them?